docs-examples/example-groovy/src/test/groovy/io/micronaut/rabbitmq/docs/headers/ProductClient.groovy

```diff
@@ -24,5 +25,9 @@ interface ProductClient {
     void send(@Header String productSize, // <3>
               @Header("x-product-count") Long count,
               byte[] data)
+
+    @Binding("product")
+    void send(RabbitHeaders rabbitHeaders, // <4>
```

Comment:
Should we show example with `MessageHeaders` interface?

Reply:
Changes to the  MessageHeaders interface.